assembl/static2/js/app/components/search.jsx

```diff
@@ -199,181 +211,231 @@ const highlightedLSOrTruncatedLS = (hit, field, locale) => {
-        className={props.bemBlocks.item('info')}
-        date={source.creation_date}
-        userId={source.creator_id}
-        userName={source.creator_name}
-      />
+const BaseHit = ({ bemBlocks, imageType, onLinkClick, title, url, renderBody, renderFooter }) => (
+  <div className={bemBlocks.item().mix(bemBlocks.container('item'))}>
+    <ImageType type={imageType} className={bemBlocks.item('imgtype')} />
+    <div className={bemBlocks.item('title')}>
+      {url ? <Link onClick={onLinkClick} to={url} dangerouslySetInnerHTML={{ __html: title }} /> : <p dangerouslySetInnerHTML={{ __html: title }} />}
     </div>
+    {renderBody && <div className={bemBlocks.item('content')}>{renderBody()}</div>}
+    <div className={bemBlocks.item('info')}>{renderFooter()}</div>
+  </div>
+);
+
+const PostHit = ({ bemBlocks, collapseSearch, locale, result }) => {
+  const source = result._source;
+  const subject = highlightedLSOrTruncatedLS(result, 'subject', locale);
+  const body = highlightedLSOrTruncatedLS(result, 'body', locale);
+  const published = {};
+  return (
+    <BaseHit
+      bemBlocks={bemBlocks}
+      imageType={result._type}
+      title={subject}
+      url={getUrl(result)}
+      onLinkClick={collapseSearch}
+      published={published}
+      renderBody={() => (
+        <React.Fragment>
+          <p dangerouslySetInnerHTML={{ __html: body }} />
+          <div>
+            <div title={I18n.t('search.like')} className="emoticon LikeSentimentOfPost" />
+            <div className="emoticonValue">{source.sentiment_counts.like}</div>
+            <div title={I18n.t('search.disagree')} className="emoticon DisagreeSentimentOfPost" />
+            <div className="emoticonValue">{source.sentiment_counts.disagree}</div>
+            <div title={I18n.t('search.dont_understand')} className="emoticon DontUnderstandSentimentOfPost" />
+            <div className="emoticonValue">{source.sentiment_counts.dont_understand}</div>
+            <div title={I18n.t('search.more_info')} className="emoticon MoreInfoSentimentOfPost" />
+            <div className="emoticonValue">{source.sentiment_counts.more_info}</div>
+          </div>
+        </React.Fragment>
+      )}
+      renderFooter={() => <PublishedInfo date={source.creation_date} userId={source.creator_id} userName={source.creator_name} />}
+    />
   );
 };
 
-const PostHit = connect(state => ({ locale: getLocale(state) }))(DumbPostHit);
-
-const DumbSynthesisHit = (props) => {
-  const locale = props.locale;
-  const source = props.result._source;
-  const ideas = highlightedLSOrTruncatedLS(props.result, 'ideas', locale);
+const DumbExtractHit = ({ bemBlocks, collapseSearch, isHarvesting, locale, toggleHarvesting, result }) => {
+  const source = result._source;
+  const subject = highlightedLSOrTruncatedLS(result, 'subject', locale);
+  const body = highlightedTextOrTruncatedText(result, 'body');
+  const onLinkClick = () => {
+    if (!isHarvesting) {
+      toggleHarvesting();
```

Comment:
Might have to disable this if this search component is on V1 interface.

Reply:
@ayyazdaniaryan can we search for extracts in v1?

Reply:
I tested search in v1 and it seems to be ok so I guess I'll merge